internal/cmd/config.go

```diff
@@ -905,4 +909,13 @@ traefik:
 	v.SetDefault("telemetry::debug", true)
 
 	v.SetDefault("integratedservice::v2", false)
+
+	// Integrated Service Operator
+	v.SetDefault("isoperator::enabled", true)
+	v.SetDefault("isoperator::repourl", "")
```

Comment:
Do we really need the url? we know we want to update the banzaicloud stable repo. If the chart would be in a different repo, we would have to add it first.